flake.nix

```
@@ -33,7 +33,7 @@
 
     devShells = flake-utils.lib.flattenTree rec {
       default = pkgs.libVanguard.overrideAttrs (attrs: {
-        buildInputs = attrs.buildInputs
+        buildInputs = attrs.propagatedBuildInputs
```

This will "delete" the `buildInputs` on the `libVanguard` derivation if they exist. The original problem can be fixed with:
```suggestion
        buildInputs = (attrs.buildInputs or [])
```